.github/workflows/linux.yml

```diff
@@ -61,5 +58,8 @@ jobs:
           cmakeVersion: 3.10.2
       - name: CMake Version
         run: cmake --version
+      - name: Create build directory
+        run: mkdir build/
       - name: Configure Vulkan-Headers
-        run: mkdir build/ && cd build/ && cmake ${GITHUB_WORKSPACE}
+        working-directory: ./build
+        run: cmake ../
```

Comment:
I'd say to just run `cmake -Bbuild`

Reply:
Just like above, apparently.

Reply:
It's just a suggestion to simplify these lines, to match what's done above; you're only configuring a cmake project here without building it (that'd be `cmake --build path/to/generated/build/folder`).

Reply:
`cmake --build ...` won't configure a project unless a it already exists. So I'm confused what you mean.

Reply:
```
juan@pop-os:~/projects/pub-headers$ ls
BUILD.gn  BUILD.md  cmake  CMakeLists.txt  CODE_OF_CONDUCT.md  CONTRIBUTING.md  include  INTEGRATION.md  LICENSE.txt  README.md  registry  tests
juan@pop-os:~/projects/pub-headers$ cmake --build build/
Error: /home/juan/projects/pub-headers/build is not a directory
```

Reply:
@juan-lunarg I said to use `cmake -B build`. `-B` **is not** a short-hand for `--build`!

Reply:
Ah. That makes more sense. Sorry for the confusion that's my bad.

Sadly CMake `3.10.2` doesn't have `-S` / `-B` they were introduced later.

Reply:
Yep `-S` and `-B` were introduced in 3.13 and we are explicitly testing the cmake_minimum advertised

Reply:
Ah that explains, was already wondering!